components/Layout/SideBar.tsx

```diff
@@ -29,13 +34,28 @@ const handlePrint = () => {
 export const SideBar: React.FC<HeadProps> = ({ data }) => {
     const { theme } = useContext(ThemeContext)
     const currentTheme = Theme[theme]
-
     const isPrintMode = useIsPrintMode()
+
+    const [isZoomIcon, setIsZoomIcon] = useState(false)
+    const resizeImg = () => {
+        if (!isZoomIcon) {
+            document.body.style.overflow = 'hidden'
+        } else {
+            document.body.style.overflow = 'inherit'
+        }
```

Comment:
This can be shorthand with `document.body.style.overflow = isZoomIcon ?  'inherit' : 'hidden'`